app/models/sketch.rb

```diff
@@ -54,14 +54,21 @@ def update_boards_metadata
         Board.find_or_create_by(board)
       end
       # clear boards metadata
-      Board.for_user(user_id).find_by(mac: board['mac']).clear_boards_metadata
+      Board.for_user(user_id).find_by(mac: board['mac'])&.clear_boards_metadata
```

Comment:
problema e atunci cand un board e sters sau deregistrat de catre un user, si sketchul inca exista. Si asta se intampla des de la laptops care atunci cand se updatau se schimbau si ramanea un board vechi in urma, iar cand il stergeam se intampla asta. Dar oricum, e bine pus acolo